packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverButton/useTeachingPopoverButtonStyles.styles.ts

```diff
@@ -1,35 +1,95 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SlotClassNames } from '@fluentui/react-utilities';
+import { useButtonStyles_unstable } from '@fluentui/react-button';
 import type { TeachingPopoverButtonSlots, TeachingPopoverButtonState } from './TeachingPopoverButton.types';
+import { makeStyles, mergeClasses } from '@griffel/react';
+import { usePopoverContext_unstable } from '@fluentui/react-popover';
+import { tokens } from '@fluentui/react-theme';
+import { useTeachingPopoverContext_unstable } from '../../TeachingPopoverContext';
+import { SlotClassNames } from '@fluentui/react-utilities';
 
 export const teachingPopoverButtonClassNames: SlotClassNames<TeachingPopoverButtonSlots> = {
   root: 'fui-TeachingPopoverButton',
-  // TODO: add class names for all slots on TeachingPopoverButtonSlots.
-  // Should be of the form `<slotName>: 'fui-TeachingPopoverButton__<slotName>`
+  icon: 'fui-TeachingPopoverButton__icon',
 };
 
-/**
- * Styles for the root slot
- */
 const useStyles = makeStyles({
-  root: {
-    // TODO Add default styles for the root element
+  brandPrimary: {
+    borderBlockColor: tokens.colorNeutralForegroundOnBrand,
+    borderInlineColor: tokens.colorNeutralForegroundOnBrand,
```

Comment:
These properties are out of our browser support matrix (https://developer.mozilla.org/en-US/docs/Web/CSS/border-inline-color). Please replace all `*Inline*` & `*Block` definitions.